es/index_settings.json

```diff
@@ -72,6 +115,22 @@
 					"letter",
 					"digit"
 				]
+			},
+			"ja_kuromoji_tokenizer": {
+				"mode": "search",
+				"type": "kuromoji_tokenizer",
+				"discard_compound_token": true,
+				"user_dictionary_rules": [
+				]
+			},
+			"ja_ngram_tokenizer": {
+				"type": "ngram",
```

Comment:
You shouldn't switch from `edge_ngram` to `ngram` here because you loose search-as-you-type functionality. The query structure is specifically set up to expect edge ngrams.

Reply:
Thanks! I also noticed this. In the latest work branch, I used `edge_ngram` for `collector.default_ja`, but I still use `ngram` for `collector.ja`.

https://github.com/MIERUNE/photon/blob/17836000db3924d5a645f1c4b5112d94e24b19be/es/index_settings.json#L25-L53

I will try to use `edge_ngram` for both `default_ja` and `collector.ja`. It may work well...